### ISSUE TITLE
Dependencies

### ISSUE THREAD

**Issue by dj8552** [ottomated/turtle-gambit#24]
What are the dependencies needed to run this

**Comment by MrEnder0** [comment]
Ok

**Comment by dj8552** [comment]
not a dependency

**Comment by Myoozik** [comment]
Yes there is, in order to install them run `npm install` in both directories

**Comment by dj8552** [comment]
> Yes there is, in order to install them run `npm install` in both directories

in which directories?
